spec/amp-html-format.md

```diff
@@ -286,6 +288,28 @@ In the following examples `<property>` needs to be in the whitelist above.
 #### Maximum size
 It is a validation error if the author stylesheet is larger than 50,000 bytes.
 
+### Keyframes stylesheet
+
+In addition to `<style amp-custom>`, authors may also add `<style amp-keyframes>` tag.
```

Comment:
Rewrite as follows:

In addition to the <style amp-custom> tag, authors may also add the <style amp-keyframes> tag, which is specifically for keyframes animations:

The following restrictions apply to the <style amp-keyframes> tag:

Reply:
Done